src/main/java/kdt/prgrms/kazedon/everevent/controller/UserController.java

```diff
@@ -40,9 +40,21 @@
     }
   }
 
-  public SignUpRequest encodingPassword(SignUpRequest request){
-    request.encodingPassword(passwordEncoder.encode(request.getPassword()));
-    return request;
+  @GetMapping("/signup/check")
+  public ResponseEntity<Void> checkDuplicate(@RequestParam String type, @RequestParam String value){
+    switch (type){
+      case "email"-> {
+        return isDuplicated(userService.checkEmailDuplicate(value));
+      }
+      case "nickname" -> {
+        return isDuplicated(userService.checkNicknameDuplicate(value));
+      }
+    }
+    return ResponseEntity.badRequest().build();
```

Comment:
회의때 얘기했던 것 처럼 중복 상황때 409 에러 반환 하는 것도 괜찮을 것 같네용~! [참고자료](https://deveric.tistory.com/62)
```suggestion
    return ResponseEntity.status(HttpStatus.CONFLICT).build(); 
```

Reply:
넵 적용하겠습니다~!